### DISCUSSION TITLE
BAE3. update or updated psi (environment):

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#192] [perspectives; sense-making]
Q1. there's difference between voluntary environment change (update phi) vs adaptation to the changed environment (updated phi). I'm curious how these differences affect the following actions and whether I can call the former expatative.  

Q2.
There's no theoretically grounded definition of pivot yet and its perception differs:
- keeping the vision, changing strategy - Ries (lean startup) and Eisenmann (Harvard prof.)
- Hierarchy of startup decision: vision-strategy-roadmap. pivot is change in the vision level” - Daniel Lee (previous Generable CTO)
- framing startup pivot as p(y|theta, phi=new)>p(y|theta, phi=current), lane changing model from #186 can be relevant

context: Charlie and I are interested in defining pivot. I wonder whether the joint inference of goal and belief mechanism detailed in [grounding language about belief in bayes theory of mind](https://arxiv.org/pdf/2402.10416v1.pdf) appear in nature.  "Baldwin effect" is the closest which I'm building up [below](https://github.com/Data4DM/BayesSD/discussions/192?sort=old#discussioncomment-8566941).
> how humans infer other agents' goals, beliefs, and plans from their actions by modeling these inferences probabilistically. This model incorporates the concept that our understanding of others' beliefs is deeply tied to the goals we infer they have and the plans they make to achieve these goals.

Q3. has there been attempts to determine the optimal level of "empty" space (where spandrels can grow) beneficial for adaptation? e.g. can it be assessed similarly to the secretary problem's 37% rule, plus how would the environment's "hardwareness" (defined as by variance ratio of demand and supply function i.e. uncertainty in data generating process of each) affect this optimal ratio?

Q4. can we draw parallels between academic theories and physical or biological entities? i.e.  hypotheses, unit theories, and programmatic theories might correspond to phenotypes, alleles, and genotypes, respectively, based on how they represent and integrate environmental understanding and observable traits.

detail below for each

**Comment by hyunjimoon** [comment]
Q1. 
Regarding Exaptation, 
- Transition from college shuttle to broader transportation system: Could be exaptation if the existing shuttle service's logistics or operational model was repurposed for a broader market. (Bridj)
- Platform expansion and monetization options: Could be exaptation if the startup repurposes existing features of the platform to create new revenue streams, such as using data analytics for targeted advertising.
- Product development to enter SaaS marketplace: Exaptation if the company pivots by leveraging existing technologies or capabilities to offer new SaaS products.
- Mission shift from flying to filming for UAVs: Exaptation, as the company is repurposing its technology for a different market (from hobbyist flying to professional filming). (DJI innovations)
- Building capabilities and market focus: Could be exaptation if the expansion is based on repurposing the existing technological capabilities for new market needs. (Evie.ai)
- Service deployment and social Wi-Fi concept: Exaptation, especially with the idea of a social Wi-Fi network, which repurposes connectivity for a new service offering. (Fourtet (Sigfox))
- Business model experimentation in hotel concept: Exaptation if the hotel rooms' standard use is repurposed for new consumption scenarios, like showrooms. (Xbed)
---
not Exaptation

- Aspirations not met due to unreliability and critical mass: Not exaptation, as it does not indicate a repurposing of existing features or technologies.
- Expansion into major cities: Not necessarily exaptation, unless the schools' expansion leverages an unintended feature of their existing system for strategic differentiation.
- Difficulty scaling in e-commerce: Not exaptation, as it seems to be a challenge rather than a strategic pivot using existing attributes in new ways.
- Challenges with scaling and funding: Not exaptation, as it relates to funding strategy rather than the innovative use of existing features or technologies.
- Financial challenges and market entry: Not exaptation, as this is a financial pivot, not an innovative repurposing of existing capabilities.
- Targeting larger retailers and cost management: Not exaptation, unless the company's approach to cost management involves repurposing existing processes or features for new strategic advantages.
- Mergers and acquisitions to become market leaders: Not exaptation, as this is a strategic business decision rather than an innovation of existing features.
- Cost reduction strategies: Not exaptation, as this is more about operational efficiency than repurposing existing capabilities.
- Acquiring tech talent and startups for scalability: Not directly exaptation, as it is about acquiring new capabilities rather than repurposing existing ones.
- Reinvestment for steady growth and regional expansion: Not exaptation, as it doesn't indicate the use of existing features in new ways.
- Business model experimentation in hotel concept: Exaptation if the hotel rooms' standard use is repurposed for new consumption scenarios, like showrooms.

**Reply by hyunjimoon** [reply]
JB explained:
exaptation theory demonstrates is that for an adaptation to occur (a response to environment pressure), there is always a pre-existing terrain. Two major ones are usually discussed (Buss 1997, Gould 1991):

First, an exaptation is "a feature, now useful to an organism, that did not arise as an adaptation for its present role, but was subsequently co-opted for its current function" 



Second, exaptations are "features that now enhance fitness, but were not built by natural selection for their current role".



So by relying on the diversity of potential technologies, products or features that it has, a startup can pivot more easily. Exaptation would ground evolutionary speaking this pivot (or critical moments in evolution) and precise the genealogy of these pivots by focusing on the features or functions that were adapted or created from scratch while pivoting.

- studied and unveiled the conditions that facilitate these transitions (pivots) that I call « creative reconfigurations » and examined their exaptive nature. For example, the importance of empty spaces and times where byproducts can grow (spandrels). Practically speaking, exaptation theory can give precise understanding of the pivoting moments in evolution as well as the epigenetic conditions favorable to them.

**Reply by hyunjimoon** [reply]
chatted with shrivm on nucleo-base < allel < gene ; different evironment, pleotropy, microbiome, nature selects for the organism to have feature for effect1

**Reply by chasfine** [reply]
I think the creation of Amazon Web Services is perhaps a good example of exaptation.  There is a HBS case on this form about 10-12 years ago.

**Reply by hyunjimoon** [reply]
I wonder whether there are counterfactual simulation studies in biology on "what if certain exaptation didn't occur?" to measure its effect. given there exist established methodology perhaps we can try to measure pure effect of evolution to AWS using below finance from https://www.economist.com/briefing/2020/06/18/can-amazon-keep-growing-like-a-youthful-startup - 
<img width="400" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/7955f2dd-8017-4d98-b77f-c9398d3078e1">

**Comment by hyunjimoon** [comment]
Q2. example of Baldwin effect

Learning to tolerate new foods: In a population where food sources change, individuals that can learn to eat new foods may survive better. Over generations, the ability to metabolize these new foods might become genetically encoded in the species, reflecting an evolutionary shift prompted by learned behavior.

Migration patterns: Some birds learn new migration routes in response to environmental changes. Over time, these new routes can become genetically programmed, making the learned behavior an inherited one.

Predator avoidance: Animals that learn to avoid predators more effectively can pass on these behaviors through teaching or through the selection of traits that facilitate such learning. Eventually, these learned strategies could influence the development of instinctual behaviors encoded in the species' genetics.

These examples illustrate how behaviors initially learned in response to environmental challenges can influence genetic evolution

**Comment by hyunjimoon** [comment]
Q3, Q4
based on the need-sol-method-delivery supply chain from https://github.com/Data4DM/BayesSD/discussions/190#discussioncomment-8672261, 
![image](https://github.com/Data4DM/BayesSD/assets/30194633/95a59179-30b8-42ea-a18c-6095dea4aae7)

**Comment by hyunjimoon** [comment]
@jeanbaptiste

<img width="701" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/bb5fef7b-0723-4cfc-b3f4-9ad79b72d197">

Lots of pivot in [Camuffo24.pdf](https://github.com/Data4DM/BayesSD/files/14727450/Camuffo24.pdf)

<img width="742" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/fd99cadd-ab1e-4ada-90a8-e58f84404dd1">

**Comment by hyunjimoon** [comment]
![16722C92-1B12-4EFA-B05F-E1790D74BEE4_1_105_c](https://github.com/user-attachments/assets/98243db4-81be-4df6-9652-2495c5ecffea)

Above diagram is a blueprint for connecting entrepreneurial learning (el) and machine learning (ml).  It illustrates a framework connecting entrepreneurial functions with different entrepreneurial approaches and machine learning paradigms. It depicts the entrepreneur as an agent engaging in a sequence of functions: perceive, approximate, experiment, strategize, persuade, act, and pivot. These functions are associated with different entrepreneurial approaches: bounded rational and effectuation (linked to behavioral approaches), scientific and theoretical (connected to Bayesian approaches), and probabilistic reasoning and pivoting (related to evolutionary approaches). The diagram then maps these entrepreneurial approaches to five machine learning paradigms: Analogizer, Connectionist, Symbolist, Bayesian, and Evolutionist. This mapping suggests parallels between entrepreneurial thinking and machine learning methods. For instance, the behavioral approach in entrepreneurship is linked to Analogizer and Connectionist learning, the Bayesian approach to Symbolist and Bayesian learning, and the evolutionary approach to Evolutionist learning. This framework provides a unified view of entrepreneurial decision-making processes and their potential computational analogues, offering insights into how different aspects of entrepreneurship might be modeled or understood through the lens of machine learning approaches, each with representing, evaluating, optimizing algorithms.

I argue machine learning (ML) convergence can contribute to entrepreneurial learning (EL) convergence, which would have positive outcome:

| ML Convergence Aspect | Contribution to EL Convergence | Potential Outcome |
|------------------------|--------------------------------|-------------------|
| Unified representation | Combine symbolic and probabilistic reasoning | Bridge different EL approaches |
| Multi-level optimization | Integrate decision-making at individual, firm, and industry levels | Reconcile behavioral, Bayesian, and evolutionary approaches |
| Adaptive learning mechanisms | Develop flexible models that switch between strategies based on context | Explain entrepreneurial adaptation across venture stages |
| Explainable AI | Make EL models more interpretable | Increase predictability of entrepreneurial outcomes |
| Computational simulations | Create sophisticated models of entrepreneurial ecosystems | Test interaction of different learning approaches |

---

I projected [Felin](https://sites.google.com/view/teppofelin/home) (I'll attend his AOM conference talk on [this](https://cdmcd.co/D86gvq), [this](https://cdmcd.co/WypZ7D), [this](https://cdmcd.co/JQJLvk))'s research on this framework

### 👁️seeing (perception, vision), rationality, cognition
Felin T & Koenderink J (2022). A generative view of rationality and growing awareness
- Perception and cognition: From omniscience to bounded rationality
- Bounded rationality and perception "specification of boundedness itself", "organisms (whether animals or humans) are not aware of, nor do they perceive or have time to compute, all alternatives in their environments", universal cognition, in seeking to develop a Bunitary theory of mind focused on external representation and the mind as a production system", "Perception and rationality: So what?" explains two different conceptions of human nature and rationality 1) errors and mistakes are the critical phenomena to be demonstrated and explained (cf. Krueger & Funder, 2004). This literature uses the norm of omniscience as a convenient null hypothesis 2) focuses not on mistakes and error (from some omniscient norm), but on the nature of rationality itself. Such a theory needs to capture the accuracy manifest in human judgment (Jussim, 2015), as well as the fact that many of the seeming biases have heuristic value and lead to better judgments and outcomes

Chater N, Felin T, et al. (2019). Mind, rationality and cognition: An interdisciplinary debate. Psychonomic Bulletin & Review.
Felin T, Koenderink J & Krueger J (2017). Rationality, perception, and the all-seeing eye. Psychonomic Bulletin & Review.

### 🧠scientific/theorist approach

Theory Is All You Need: AI, Human Cognition, and Decision Making 
https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4737265
From "DISCUSSION: THE LIMITS OF PREDICTION FOR DECISION MAKING UNDER UNCERTAINTY" section, if we replace "prediction" with "prediction under intervention in yet unseen situation", would this include the human's role (theorizing, artist (imagine what might happen)'s in Matt's word) described in the paper?


### 🤜evolution

Felin T & Kauffman S (2023). Disruptive evolution: Harnessing functional excess, experimentation and science as tool. Industrial and Corporate Change.
Felin T & Kauffman S (2021). The search function and evolutionary novelty. Oxford University Press.
Felin T, Kauffman S, Koppl R & Longo G (2014). Economic opportunity and evolution: Beyond landscapes and bounded rationality. Strategic Entrepreneurship Journal.
- sec.PHASE SPACE AND LANDSCAPE defines phase space as "representation of theset of all possible actions, strategies, or states"; in biological evolution, phase space cannot be speciﬁed even on average
- notion of development, as the counterpart of selection, suggests that there are possible trajectories, or ‘evol-vability’
- there are some, more narrow, settings where it is possible to list allavailable options or to simulate the search foroptimal outcomes. But, the context of entrepreneurial and strategic activity is not one of these settings

---
Thanks @jeanbaptiste for input on Stuart Kauffman, @chasfine for input on Toward a Theory of the Evolution of Business Ecosystems by Piepenbrock and Fine (2009)
